src/status_im/contexts/syncing/device/view.cljs

```diff
@@ -1,17 +1,47 @@
 (ns status-im.contexts.syncing.device.view
   (:require
     [quo.core :as quo]
+    [react-native.core :as rn]
     [status-im.contexts.syncing.device.style :as style]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- hide-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
```

Comment:
We don't need this private function, We can just have an anonymous function on line 28.

Reply:
We're trying to avoid inlining anonymous functions as props so I think it's also fine as a private function so that in case the component re-renders it won't be re-created. It also takes no args, so I think it belongs outside the component.